src/main/java/util/events/testevents/UpdateDummyRepoEvent.java

```diff
@@ -25,43 +27,50 @@
      * @param itemId The id of the element to be updated or deleted, if necessary.
      * @param updateText The text to update the element with, if necessary.
      */
-    public UpdateDummyRepoEvent(UpdateType updateType, String repoId, int itemId, String idString, String updateText) {
+    public UpdateDummyRepoEvent(UpdateType updateType, String repoId, int itemId, String idString, String updateText,
+                                String actor) {
         this.updateType = updateType;
         this.repoId = repoId;
         this.itemId = itemId;
         this.idString = idString;
         this.updateText = updateText;
+        this.actor = actor;
     }
 
     // Overloaded constructors
 
     // Hardly used, maybe to clean up entire model
     public UpdateDummyRepoEvent(UpdateType updateType) {
-        this(updateType, null, -1, null, null);
+        this(updateType, null, -1, null, null, "");
     }
 
     // e.g. Clean repo, new issue
     public UpdateDummyRepoEvent(UpdateType updateType, String repoId) {
-        this(updateType, repoId, -1, null, null);
+        this(updateType, repoId, -1, null, null, "");
     }
 
-    // e.g. Delete issue
+    // e.g. Delete issue, delete milestone
     public UpdateDummyRepoEvent(UpdateType updateType, String repoId, int itemId) {
-        this(updateType, repoId, itemId, null, null);
+        this(updateType, repoId, itemId, null, null, "");
     }
 
     // e.g. Delete label
     public UpdateDummyRepoEvent(UpdateType updateType, String repoId, String idString) {
-        this(updateType, repoId, -1, idString, null);
+        this(updateType, repoId, -1, idString, null, "");
     }
 
-    // e.g. Update issue
+    // e.g. Update issue, update milestone
     public UpdateDummyRepoEvent(UpdateType updateType, String repoId, int itemId, String updateText) {
-        this(updateType, repoId, itemId, null, updateText);
+        this(updateType, repoId, itemId, null, updateText, "");
     }
 
     // e.g. Update label
     public UpdateDummyRepoEvent(UpdateType updateType, String repoId, String idString, String updateText) {
-        this(updateType, repoId, -1, idString, updateText);
+        this(updateType, repoId, -1, idString, updateText, "");
+    }
+
+    // e.g. Comment
+    public UpdateDummyRepoEvent(UpdateType updateType, String repoId, int itemId, String updateText, String actor) {
+        this(updateType, repoId, itemId, null, updateText, actor);
     }
```

Comment:
This can be replaced with factory methods, like how `Optional.of` and `Optional.empty` are used in lieu of constructors. They are more self-documenting and less error-prone, especially in cases like this, where we have many ways to construct the same thing.

How about something like `UpdateDummyRepoEvent.forLabel`, `UpdateDummyRepoEvent.forMilestone`, etc.? Can use a static import if this starts getting verbose.

`UpdateType` could even be an interval value, not exposed to users. If there's no need for a default constructor, it can be made private as well.